modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -35,13 +35,13 @@ def cancel
       private
 
       def fetch_cached_or_service(validated_params)
-        json = Mobile::V0::Appointment.get_cached_appointments(@current_user) if validated_params[:use_cache]
+        json = nil
+        json = Mobile::V0::Appointment.get_cached(@current_user) if validated_params[:use_cache]
 
         # if JSON has been retrieved from redis, delete the cached version and return recovered appointments
         # otherwise fetch appointments from the upstream service
         if json
           Rails.logger.info('mobile appointments cache fetch', user_uuid: @current_user.uuid)
-          Mobile::V0::Appointment.delete_cached_appointments(@current_user)
```

Comment:
This is intended. Originally this was added so the cache was only available the first time the appointments screen was hit but 1) it's confusing it works that way and 2) API users have control of if they want to hit the cache or not via the `useCache` query param.